common/models/acl.js

```diff
@@ -250,9 +252,12 @@ module.exports = function(ACL) {
         debug('with score:', acl.score(req));
       });
     }
-
-    var res = new AccessRequest(req.model, req.property, req.accessType,
-        permission || ACL.DEFAULT);
+    var res = new AccessRequest({
+      model: req.model,
+      property: req.property,
+      accessType: req.accessType,
+      permission: permission || ACL.DEFAULT,
+      registry: this.registry});
     return res;
```

Comment:
switching to one object param declaration when adding the registry as it reduces chances for further errors (adding things like `(param1, param2, null, null, param5)`)

Note: this should the default documented option, as for AccessContext